src/propOr.js

```diff
@@ -28,5 +29,5 @@ var _has = require('./internal/_has');
  *      favoriteWithDefault(alice);  //=> 'Ramda'
  */
 module.exports = _curry3(function propOr(val, p, obj) {
-  return _has(p, obj) ? obj[p] : val;
+  return (!isNil(obj) && _has(p, obj)) ? obj[p] : val;
```

Comment:
All things are relative.

Uncle Albert (Einstein) was always muttering something about that.